spec/postgres_admin_spec.rb

```diff
@@ -21,28 +22,29 @@
       end
     end
 
-    context "with a pg_dump file from a pipe" do
-      let(:dbname)    { "pg_dump_restore_of_simple_db_from_pipe" }
-      let(:fifo_path) { Pathname.new(Dir::Tmpname.create("") {}) }
+    # context "with a pg_dump file from a pipe" do
```

Comment:
Do we need this anymore?

Reply:
Probably not... it was failing, but there is no real reason that it should though.

But it isn't really needed, so it can be tossed.  Will do that in a follow up.